test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -12,14 +25,48 @@ public void constructor_loadsSettings() throws SettingsLoadException {
   }
 
   @Test
-  public void main_noHelp_canExecuteWithoutCrashing() throws SettingsLoadException {
+  public void main_noHelp_callsExecutFromCmdLine() throws Exception {
+    WasapiDownloader mockDownloader = PowerMockito.mock(WasapiDownloader.class);
+    PowerMockito.whenNew(WasapiDownloader.class).withAnyArguments().thenReturn(mockDownloader);
+
     WasapiDownloader.main(null);
+    verify(mockDownloader).executeFromCmdLine();
   }
 
   @Test
   @SuppressWarnings("checkstyle:NoWhitespaceAfter")
-  public void main_withHelp_canExecuteWithoutCrashing() throws SettingsLoadException {
+  public void main_withHelp_canExecuteWithoutCrashing() throws SettingsLoadException, IOException {
     String[] args = { "-h" };
     WasapiDownloader.main(args);
   }
+
+  @Test
+  public void executeFromCmdLine_requestsFileListResponse() throws Exception {
+    WasapiConnection mockConn = Mockito.mock(WasapiConnection.class);
+    Mockito.when(mockConn.jsonQuery(anyString())).thenReturn(null);
+    WasapiDownloader mockDownloader = Mockito.spy(new WasapiDownloader(WasapiDownloader.SETTINGS_FILE_LOCATION, null));
```

Comment:
would slightly prefer to call it `spyDownloader`, but i don't feel strongly about it.